admissions/admissions/views.py

```diff
@@ -97,13 +98,15 @@ def get_queryset(self):
         user.__class__ = LegoUser
         if not user.is_privileged:
             return User.objects.none()
-        if user.is_superuser:
-            return (
-                super()
-                .get_queryset()
-                .filter(admission__slug=admission_slug)
-                .prefetch_related("group_applications", "group_applications__group")
-            )
+        admission = Admission.objects.get(slug=admission_slug)
+        for group in admission.admin_groups.all():
+            if Membership.objects.filter(user=user, group=group.pk).exists():
+                return (
+                    super()
+                    .get_queryset()
+                    .filter(admission__slug=admission_slug)
+                    .prefetch_related("group_applications", "group_applications__group")
+                )
```

Comment:
Strictly speaking not relevant to this PR, but it was a bug left behind from earlier that prevented me from being able to test the new functionality.  
Previously is_superuser defined whether a user could view all applications, but now that is defined on an application level through the `admin_groups` property